src/test/kotlin/no/nav/familie/ef/søknad/mapper/SøkerinfoMapperTest.kt

```diff
@@ -93,7 +94,7 @@ internal class SøkerinfoMapperTest {
 
     @Test
     internal fun `ikke feile når henting av poststed feiler`() {
-        every { kodeverkService.hentPoststedFor(any()) } throws RuntimeException("Feil")
+        every { kodeverkService.hentPoststed(any()) } throws RuntimeException("Feil")
```

Comment:
Kunne vi hatt en test for land som gjør dette?

Reply:
Det er gjort og fant en feil om att cachen ikke blir brukt når, man i en bøna, gjør slik, dvs en metod uten Cacheable kaller en metode med cacheable

```kotlin
fun foo() {bar()}

@Cacheable fun bar(){...}
```